test/lib/processors/stringReplacer.js

```diff
@@ -1,39 +1,46 @@
 import test from "ava";
-import {Readable} from "node:stream";
+import _sinon from "sinon";
```

Comment:
is it necessary to use here "_" ? The const declarations in the the tests are created in a new scope isn't?

Reply:
Ack.